src/easier68k/simulator/m68k.py

```diff
@@ -161,7 +162,26 @@ def step_instruction(self):
         counter increments
         :return:
         """
-        pass
+        # must be here or we get circular dependency issues
+        from ..core.opcodes.move import from_binary
+        from ..core.util.find_module import find_module, valid_opcodes
+
+        for op_str in valid_opcodes:
+            op_module, op_class = find_module(op_str)
+
+            # We don't know this opcode, there's no module for it
+            if op_module is None:
+                print('Opcode {} is not known: skipping and continuing'.format(op_str))
+                continue
+
+            PC = self.get_program_counter_value()
+            # fixme: fix the hax. 20 is plenty
+            op, n = op_module.from_binary(self.memory.memory[PC:PC+20])
```

Comment:
fix this?

Reply:
Oops. I forgot about that :) Helps to read self-reminders